doc/devel/contributing.rst

```diff
@@ -590,3 +590,64 @@ example code you can load it into a file handle with::
 
     import matplotlib.cbook as cbook
     fh = cbook.get_sample_data('mydata.dat')
+
+
+Baseline image generation
+-------------------------
+The idea is to eventually remove all of the images but we have options about where devs get the baselines from.
```

Comment:
This needs a paragraph explaining the current state of things to the reader.  To us (who have been thinking about this all summer), this makes perfect sense, but to a new contributor who does not even know we _have_ baseline images checked into the repo this is going to be consufing.

Reply:
Done